services/src/kilda-pce/src/main/java/org/openkilda/pce/finder/PathFinder.java

```diff
@@ -39,6 +39,16 @@
                                                    WeightFunction weightFunction)
             throws UnroutableFlowException;
 
+    /**
+     * Finds a path whose weight is less than maxWeight and as close to maxWeight as possible.
+     *
+     * @return an ordered list that represents the path from start to end, or an empty list if no path found.
```

Comment:
Pair of ordered lists that represents forward path from start to end and reverse one

Correct javadoc for previous method also, please.

Reply:
Fixed.